tensorboard/plugins/audio/summary_test.py

```diff
@@ -250,6 +250,39 @@ def test_default_step(self):
             # Reset to default state for other tests.
             tf2.summary.experimental.set_step(None)
 
+    def test_lengths(self):
```

Comment:
Could you add a test to cover the case where `max_outputs` actually
limits the batch size? (i.e., `len(lengths) > max_outputs`)

Reply:
Done -- if you don't mind I just added max_outputs to the existing 2 test cases I added :).